spec/classes/init_spec.rb

```
@@ -196,6 +196,12 @@
               }
             else
               it {
+                # Stub /tmp_docker dir to prevent shelling out during spec test
```

My guess is that these are hiding real errors.

---

This stub/mock is to address the spec test only, not acceptance tests (which passed)